ios/FluentUI.Demo/FluentUI.Demo/Demos/ThemingDemoController.swift

```diff
@@ -153,3 +116,77 @@ class ThemingDemoController: DemoController {
         }
     }
 }
+
+open class CustomStyle: FluentUIStyle {
```

Comment:
This doesn't seem to be autogenerated, do we need any manual updates to this wehn when we add a new design token?

Reply:
No. It's not autogenerated. This is a demonstration on how to override the new classes with their read only properties.